.gitignore

```diff
@@ -18,6 +18,7 @@ docs/source/demos
 docs/source/_build/
```

Comment:
As discussed - we will wait until Jesse and Alec have merged their PRs before merging this

Reply:
#7786 is merged now, that's the important one, so I think let's proceed.

Reply:
Finally I didn't merge this as a test merge (not even a rebase) of the branch into [experiment-session-deprecation](https://github.com/Dallinger/Dallinger/tree/experiment-session-deprecation) (PR https://github.com/Dallinger/Dallinger/pull/7896) caused a lot of conflicts. I thought it would be better not to waste Alec's time to resolve the conflists and rather postpone this one.

Reply:
Well spotted, I missed this because it wasn't listed in the Working Project. Added now. Thanks!